web/templates/account/edit_security.html.eex

```diff
@@ -0,0 +1,38 @@
+<div class="ma3 ma4-m ma4-l mv2 mv3-m mv4-l">
+  <p>Please confirm your security details and enter new ones.</p>
+
+  <p>Current security question : </p>
+  <%= @user.security_question %>
+
+
+  <%= form_for @changeset, @action, fn f -> %>
+    <input name="_method" type="hidden" value="put" />
+    <div class="form-group">
+      <%= label :user, "Current answer", class: "control-label" %>
+      <%= text_input :user, :security_check, class: "form-control" %>
+      <%= error_tag f, :security_check %>
+    </div>
+
+    <div class="form-group">
+      <%= label f, "New security question", class: "control-label" %>
+      <%= select f, :security_question,
+        ["What were the last four digits of your childhood telephone number?": "What were the last four digits of your childhood telephone number?",
```

Comment:
I'd prefer to have these security questions in a dedicated file rather than in the template.